Windows: Unable to build SwiftGodot using Swift 5.10 toolchain

Building SwiftGodot on Windows using Swift 5.10 fails during generator plugin execution:
```
error: failed: PrebuildCommand(configuration: SPMBuildCore.BuildToolPluginInvocationResult.CommandConfiguration(displayName: Optional("Generating Swift API from /Projects/SwiftGodot/Sources/ExtensionApi/extension_api.json to /Projects/SwiftGodot/.build/plugins/outputs/swiftgodot/SwiftGodot/CodeGeneratorPlugin/GeneratedSources"), executable: <AbsolutePath:"\Projects\SwiftGodot\.build\plugins\tools\debug\Generator.exe">, arguments: ["/Projects/SwiftGodot/Sources/ExtensionApi/extension_api.json", "/Projects/SwiftGodot/.build/plugins/outputs/swiftgodot/SwiftGodot/CodeGeneratorPlugin/GeneratedSources", "/Projects/SwiftGodot/doc", "--singlefile"], environment: [:], workingDirectory: nil), outputFilesDirectory: <AbsolutePath:"\Projects\SwiftGodot\.build\plugins\outputs\swiftgodot\SwiftGodot\CodeGeneratorPlugin\GeneratedSources">)

Generator/main.swift:30: Fatal error: 'try!' expression unexpectedly raised an error: Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"
Current stack trace:
0    (null)                             0x00007ffac5964ae0 swift_stdlib_reportFatalErrorInFile + 132
```

After fixing extension api reading Windows build still fails throwing a whole bunch of errors like
```
C:\Projects\SwiftGodot\Sources\SwiftGodot\Core\VariantCollection.swift:50:24: error: cannot find 'GArray' in scope
        if let array = GArray (variant) {
                       ^~~~~~
C:\Projects\SwiftGodot\Sources\SwiftGodot\Core\VariantRepresentable.swift:91:11: error: cannot find type 'Vector2' in scope
extension Vector2: SelfVariantRepresentable {
          ^~~~~~~
C:\Projects\SwiftGodot\Sources\SwiftGodot\Core\VariantRepresentable.swift:95:11: error: cannot find type 'Vector2i' in scope
extension Vector2i: SelfVariantRepresentable {
          ^~~~~~~~
C:\Projects\SwiftGodot\Sources\SwiftGodot\Core\VariantRepresentable.swift:99:11: error: cannot find type 'Rect2' in scope
extension Rect2: SelfVariantRepresentable {
          ^~~~~
```
and so on, basically it does not include generated sources into the build for some reason.
I'd guess that `prebuildCommand` workaround does not work anymore. Need to test it on a simpler package.

--- a/Plugins/CodeGeneratorPlugin/plugin.swift
+++ b/Plugins/CodeGeneratorPlugin/plugin.swift
@@ -37,12 +37,15 @@ import PackagePlugin
         outputFiles.append (contentsOf: known.map { genSourcesDir.appending(["generated", $0])})
         #endif
 
+        // For Windows with Swift 5.10 both prebuildCommand and buildCommand are needed
+        #if !os(Windows) || swift(>=5.10)
         commands.append(Command.buildCommand(
             displayName: "Generating Swift API from \(api) to \(genSourcesDir)",
             executable: generator,
             arguments: arguments,
             inputFiles: [api],
             outputFiles: outputFiles))
+        #endif
         
         return commands
     }

--- a/Plugins/CodeGeneratorPlugin/plugin.swift
+++ b/Plugins/CodeGeneratorPlugin/plugin.swift
@@ -11,6 +11,7 @@ import PackagePlugin
 /// Generates the API for the SwiftGodot from the Godot exported Json API
 @main struct SwiftCodeGeneratorPlugin: BuildToolPlugin {
     func createBuildCommands(context: PluginContext, target: Target) throws -> [Command] {
+        var commands: [Command] = []
         // Configure the commands to write to a "GeneratedSources" directory.
         let genSourcesDir = context.pluginWorkDirectory.appending("GeneratedSources")
 
@@ -26,23 +27,24 @@ import PackagePlugin
         outputFiles.append(genSourcesDir.appending(subpath: "Generated.swift"))
         arguments.append(context.package.directory.appending(subpath: "doc"))
         arguments.append("--singlefile")
-        let cmd: Command = Command.prebuildCommand(
+        commands.append(Command.prebuildCommand(
             displayName: "Generating Swift API from \(api) to \(genSourcesDir)",
             executable: generator,
             arguments: arguments,
-            outputFilesDirectory: genSourcesDir)
+            outputFilesDirectory: genSourcesDir))
         #else
         outputFiles.append (contentsOf: knownBuiltin.map { genSourcesDir.appending(["generated-builtin", $0])})
         outputFiles.append (contentsOf: known.map { genSourcesDir.appending(["generated", $0])})
-        let cmd: Command = Command.buildCommand(
+        #endif
+
+        commands.append(Command.buildCommand(
             displayName: "Generating Swift API from \(api) to \(genSourcesDir)",
             executable: generator,
             arguments: arguments,
             inputFiles: [api],
-            outputFiles: outputFiles)
-        #endif
+            outputFiles: outputFiles))
         
-        return [cmd]
+        return commands
     }
 }
 

